src/rp2/plugin/report/es/open_positions.py

```diff
@@ -0,0 +1,544 @@
+# Copyright 2023 Larision
```

Comment:
I think this file can be deleted (see my other comment above): you should be able to use the common open_positions.py file.

Reply:
Checking about it, all languages showing $ in currency using common open_positions.
Thats why i added ES specific template_open_positions_es.ods and es.open_positions.py.

Reply:
Oh, I see: let me take a look at this later today.

Reply:
OK, I fixed the issue: if you get the sync your branch to the latest version you'll be pick up the fix (see my other comment on how to do that).